chapters/ch23.md

```diff
@@ -1,36 +1,35 @@
 # 23. Addressing Bias and Variance
--> 
-
+->
+Giải quyết Độ chệch và Phương sai
 
 Here is the simplest formula for addressing bias and variance issues:
 ->
-
+Đây là công thức đơn giản nhất để giải quyết các vấn đề độ chệch và phương sai:
 
 * If you have high avoidable bias, increase the size of your model (for example, increase the size of your neural network by adding layers/neurons).
 ->
-
+* Nếu bạn có độ chệch cao, hãy tăng kích thước mô hình của bạn (ví dụ: tăng kích thước của mạng neural bằng cách thêm các lớp/neurons)
 
 * If you have high variance, add data to your training set.
--> 
-
+->
+* Nếu bạn có phương sai cao, hãy thêm dữ liệu vào tập huấn luyện của bạn.
 
 If you are able to increase the neural network size and increase training data without limit, it is possible to do very well on many learning problems.
 ->
-
+Nếu bạn có thể không giới hạn tăng kích thước mạng neural và dữ liệu huấn luyện thì có thể xử lý rất tốt trong rất nhiều những vấn đề của machine learning.
```

Comment:
Đoạn này mình thấy nó kỳ kỳ, nếu dịch nó như thế này thì sao nhỉ?
> Nếu bạn có thể tăng kích thước của mạng neural và dữ liệu huấn luyện lên một cách không giới hạn, có thể nó sẽ xử lý rất tốt trên nhiều vấn đề của Machine Leanring.

Reply:
@seanphan @duythanhvn :
trong câu này có vẻ như "machine learning problems" nói đến "một số bài toán machine learning/bài toán giải bằng machine learning": 
- có cụm từ do well = làm tốt
- nếu như "problems" dịch là "vấn đề"  (vấn đề của machine learning) thì cụm "xử lý rất tốt trên" nên được sửa thành "giải quyết" . Anw, cách hiểu này mình thấy không fit với context của chương này.

Reply:
Thực ra trong câu này không có chữ machine mà mình cũng chưa biết dịch cụm "many learning problems" như thế nào nên add thêm cái chữ machine learning.

@duythanhvn Câu bạn chỉnh lại đọc thuận hơn nhiều. 
@william-vu Mình gom ý 2 bạn và update thành  "Nếu bạn có thể tăng kích thước của mạng neural và dữ liệu huấn luyện lên một cách không giới hạn thì bạn sẽ có khả năng xử lý rất tốt trên nhiều bài toán ứng dụng của Machine Learning."
Không word-to-word lắm nhưng mình thấy cũng ok. Mọi nguời thấy sao?

Reply:
mình ok, quan trọng là mọi người synchronize cách hiểu ở đây.